src/app/about/about.component.html

```diff
@@ -1,19 +1,18 @@
+
 <div class="container">
-  <div class="row">
-    <h1 class="center-align text-black">We the Coders</h1>
-  </div>
-  <div class="row">
-    <div id="map">
-      <img class="responsive-img" src="https://maps.googleapis.com/maps/api/staticmap?center=20,-20&zoom=1&scale=2&size=1920x225&maptype=roadmap&key=AIzaSyATPQm0vWt-TwYMgPgb8LQtwpr2xtgni1g&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CNorth+America&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CSouth+America&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CAfrica&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CEurope&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CAsia&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CAustralia" alt="Code for Social Good Volunteer Locations">
+  <div class="row main-row">
+    <div id="left-col"></div>
+    <div class="col m12" id="right-col">
+      <div class="content" *ngIf="developers">
+        <div class="row">
+          <h1 class="main-title">We, the people, are {{developers.length}} devs</h1>
```

Comment:
This About Us page is about us, who build C4SG site. 

It pulls contributors from Github, mark their locations on the map, and display their avatars linking to their user profiles.

Less is more, should we 
- simply put "We the Coders" or "We the C4SG Coders"? Or 
- a more descriptive is better such as "Meet the developers of Code for Social Good"? 
Let russon77 make the final call since his English is the best among us :)